From b5572db5454268ceff8f8d71c9c46e8f1f0c2039 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 5 Jun 2015 18:43:36 +0800
Subject: [PATCH] MLK-11056: ARM: dts: change the PAD setting for MQS

Change the PAD setting for MQS for reduce the noise

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts  | 4 ++--
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index 41abfd3..d77531c 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -529,8 +529,8 @@
 
 		pinctrl_mqs: mqsgrp {
 			fsl,pins = <
-				MX6UL_PAD_JTAG_TDI__MQS_LEFT         0x4000007f
-				MX6UL_PAD_JTAG_TDO__MQS_RIGHT        0x4000007f
+				MX6UL_PAD_JTAG_TDI__MQS_LEFT         0x11088
+				MX6UL_PAD_JTAG_TDO__MQS_RIGHT        0x11088
 			>;
 		};
 
diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 473e68c..279c631 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -665,8 +665,8 @@
 	mqs {
 		pinctrl_mqs: mqsgrp {
 			fsl,pins = <
-				MX7D_PAD_SAI1_RX_SYNC__MQS_RIGHT     0x4000007f
-				MX7D_PAD_SAI1_RX_BCLK__MQS_LEFT      0x4000007f
+				MX7D_PAD_SAI1_RX_SYNC__MQS_RIGHT     0x0
+				MX7D_PAD_SAI1_RX_BCLK__MQS_LEFT      0x0
 			>;
 		};
 	};
-- 
1.8.0

